stats/stats.go

```diff
@@ -0,0 +1,85 @@
+package stats
+
+import "time"
+
+// RPCStats contains stats information about RPCs.
+type RPCStats interface {
+	isRPCStats()
```

Comment:
doesn't seem used?

Reply:
I think this is a safe-guard from grpc to differentiate between `IsClient` interfaces. I shall remove it in our case.